etc/profile-a-l/device-flasher.linux.profile

```diff
@@ -0,0 +1,81 @@
+# Firejail profile for device-flasher.linux
+# Description: CalyxOS' device flasher
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include device-flasher.linux.local
+# Persistent global definitions
+include globals.local
+
+# Usage: Run firejail ./device-flasher.linux in the folder with it and your
+# factory image.
+
+# Warning: ADB does not work with this profile, so you need to reboot the
+# device into bootloader mode first. To do this, hold Volume Down + Power until
+# you get into a screen with an Android robot being repaired. You'll need OEM
+# Unlocking beforehand.
+
+# Fedora: You may need to manually install a third-party resource for udev
+# rules, as Fedora ships broken ones on android-tools.
+
+ignore noexec ${HOME}
+
+# Allow /bin/sh (blacklisted by disable-shell.inc)
+include allow-bin-sh.inc
+
+blacklist ${RUNUSER}
+blacklist /opt
+blacklist /srv
+blacklist /usr/libexec
+
+include disable-common.inc
+include disable-devel.inc
+include disable-exec.inc
+include disable-interpreters.inc
+include disable-proc.inc
+include disable-programs.inc
+include disable-shell.inc
+include disable-x11.inc
+include disable-xdg.inc
+
+whitelist ${DOWNLOADS}
+
+include whitelist-common.inc
```

Comment:
```suggestion
```

> I don't see much reason to use whitelist-common.inc, it isn't needed at all,
> especially being a CLI tool.

After looking through whitelist-common.inc again, I think it makes sense to
leave it out, as almost all paths are for GUI programs.

Misc: Maybe it would make sense to eventually rename it to
whitelist-gui-common.inc.